packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -268,6 +268,24 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
                 }
             }
         }));
+
+        toDispose.push(this.commands.registerCommand({
+            id: `${view.id}.focus`,
+            label: `Focus ${view.name} View`
```

Comment:
Are you sure that such commands should have UI contributions. I could not find anything in VS Code, for us it is a bit overkill, since we have other commands. Is it for testing?

Reply:
Vscode registeres this command for every registered view:
https://github.com/microsoft/vscode/blob/master/src/vs/workbench/browser/parts/views/views.ts#L542

vscode-references-view plugin uses it to reveal reference view, and probably some other plugins do. So, for sure the plugin needs it and it's not a test command.

Reply:
I mean if we add `label` then this command is visible in UI like in the command palette. I cannot find such commands visible in VS Code. That we register a command is fine me.

Reply:
Yes, no problem. I have removed `label` property. Now the command is not visible and the plugin works fine with this.